### ISSUE TITLE
Use JOSA's default avatar instead of the alt text

### ISSUE THREAD

**Issue by alsahawneh** [jordanopensource/community-web#129]
When the member's avatar fails to load, it's better to show the default avatar instead of an alt text. 

![Screenshot 2022-11-23 at 18-59-28 JOSA Community](https://user-images.githubusercontent.com/79845228/203592615-7d4ab00a-cc9c-4322-ac2d-5ed18aa7a78c.png)

**Comment by thamudi** [comment]
@itsmohmans please follow up on this task

**Comment by itsmohmans** [comment]
@alsahawneh how do you reproduce this? because the default JOSA placeholder always loads when there's no avatar by user.

**Comment by thamudi** [comment]
Closed due to it not being reproduceable
